### ISSUE TITLE
"Unable to find binary extension, was hitimes installed correctly" after installing v1.8.1

### ISSUE THREAD

**Issue by maxandersen** [mitchellh/vagrant#6801]
I updated from a working 1.7.4 to a now broken 1.8.1 via osx package on vagrantup.com.

When I run `vagrant --version` I get "Vagrant 1.8.1", but running any other command gives me:

```
Vagrant failed to initialize at a very early stage:

The plugins failed to load properly. The error message given is
shown below.

Unable to find binary extension, was hitimes installed correctly?
```

Any pointers ? I'm running El Capitain if that matters.

**Comment by aequitas** [comment]
Same here, running plugin related commands does work:

    $ vagrant --version
    Vagrant 1.8.1

    $ vagrant plugin list
    landrush (0.18.0)
    vagrant-aws (0.6.0)
    vagrant-share (1.1.5, system)
    vagrant-vbguest (0.11.0)
    vagrant-vbox-snapshot (0.0.10)

**Comment by aequitas** [comment]
Uninstalling and reinstalling plugins solved the problem for me.

**Comment by maxandersen** [comment]
a full removal of vagrant made this work again for me.

uninstalling/reinstalling plugins was not enough.

**Comment by sethvargo** [comment]
Hi everyone,

Thank you for opening and commenting on this issue, and I am sorry the upgrade process has not been smooth. Unfortunately I am not able to reproduce this issue, but I am glad you were able to get it fixed by uninstalling plugins or re-installing Vagrant.

It is possible that the `.vagrant.d` directory was somehow put in a bad state. However, without a reproduction, it's really hard for us to fix it :frowning: 

I am going to go ahead and close this issue, but if someone is able to provide a reproducible way, we can definitely get this fixed in core. Thanks! :smile:

**Comment by cswingler** [comment]
I seem to be able to reliably reproduce this:

https://gist.github.com/cswingler/cb3bbfb8d873d5f88c4e

**Comment by sethvargo** [comment]
@cswingler that error is coming from the berkshelf plugin, not Vagrant

**Comment by simonsparks** [comment]
I managed to get past this problem by uninstalling just one plugin (landrush in my case as it was first on the list).

**Comment by mario-areias** [comment]
@sethvargo I am exactly same situation as described by @maxandersen. Do you want me to get any specific logs so you can investigate?

**Comment by mario-areias** [comment]
I don't have the berkshelf plugin, btw.

**Comment by mario-areias** [comment]
This logs is when I try to `reload` a machine with `--debug` enabled

```
DEBUG global: Loading core plugin: /opt/vagrant/embedded/gems/gems/vagrant-1.8.1/plugins/synced_folders/smb/plugin.rb
 INFO manager: Registered plugin: SMB synced folders
 INFO global: Loading plugins!
 INFO manager: Registered plugin: vagrant-share
 INFO manager: Registered plugin: vagrant-vbguest
ERROR vagrant: Vagrant experienced an error! Details:
ERROR vagrant: #<Vagrant::Errors::PluginLoadError: The plugins failed to load properly. The error message given is
shown below.

Unable to find binary extension, was hitimes installed correctly?>
ERROR vagrant: The plugins failed to load properly. The error message given is
shown below.

Unable to find binary extension, was hitimes installed correctly?
ERROR vagrant: /opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant.rb:293:in `rescue in <top (required)>'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/lib/vagrant.rb:289:in `<top (required)>'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/bin/vagrant:105:in `require'
/opt/vagrant/embedded/gems/gems/vagrant-1.8.1/bin/vagrant:105:in `<main>'
Vagrant failed to initialize at a very early stage:

The plugins failed to load properly. The error message given is
shown below.

Unable to find binary extension, was hitimes installed correctly?
```

**Comment by duocgit** [comment]
Uninstalling hitimes plugin worked for me

**Comment by dentarg** [comment]
Same here uninstall of landrush solved it (only plugin I had installed myself, `vagrant plugin list` listed vagrant-share too)

**Comment by marsmensch** [comment]
Same issue here. After upgrading a fully functional Vagrant 1.7.4 with landrush plugin (0.18.0), i got the error message experienced by @maxandersen @aequitas and @mario-areias.

Uninstalling landrush and reinstalling helped to work around the problem.

**Comment by mikegreiling** [comment]
So, is landrush incompatible with 1.8.1 or is this merely an upgrade process issue?

**Comment by marsmensch** [comment]
landrush works perfectly after reinstalling it. i guess it is some existing
landrush configuration that messes things up.

**Comment by davsclaus** [comment]
Yeah I had that too in a 1.7.4 -> 1.8.1 upgrade on osx. To resolve I trashed my `~/.vagrant.d` dir, and then installed landrush again, and it now works.

**Comment by petems** [comment]
Uninstalling landrush and reinstalling fixed for me :+1:

**Comment by pbattino** [comment]
Same as petems! I had landrush 0.18.0, now it's 0.19.0
Thank you guys

**Comment by pbattino** [comment]
Correction: it SEEMED to word, but it didn't. I needed to empty  ~/.vagrant.d/data/landrush and reinstall landrush.

**Comment by cfont** [comment]
I ran into this same issue and resolved it like @petems by simply uninstalling and reinstalling landrush. In  my case I did not need to empty .vagrant.d dir but instead only reinstall landrush. I am using a Mac 10.10.4 and originally installed vagrant using homebrew in case that should be important at all. 

Something else that may be important is that I don't have hitimes plugin installed before or after upgrading vagrant.
